SIPs/SIP-000X.md

```diff
@@ -0,0 +1,164 @@
+4. Submit the new file from (2) as a pull request to the Bitocracy repo. This will be the SIP pull request. If the SIP is approved by SOV holders, the pull request created in (1) will be merged. If the proposal is rejected/withdrawn, the pull request created in (1) will be closed.
+
+### Stage IV: Review Periods
+After an SIP has been submitted as a draft pull request to the Bitocracy repo, it SHOULD undergo three review periods:
+
+1. A Community Review, which starts the moment that a proposal has been submitted to the Bitocracy repo and lasts for one calendar week.  
+2. An Editor Review, which starts when the Community Review ends and lasts for an indefinite period of time.  
+3. A Final Review, which starts when the Editor Review ends and lasts for one calendar week.  
+
+**Community Review**  
+During the Community Review period, the SIP author will have a chance to respond to feedback and make changes to their proposal based on the feedback they have received to increase the likelihood of the proposal passing.  
+
+**Editor Review**  
+At the end of the Community Review period, SIP Editors will perform their review of the proposal. The SIP author SHOULD incorporate any changes suggested by the SIP Editor to prepare the proposal for the Final Review. After a proposal is reviewed by SIP Editors, an SIP Editor will ask the author if the proposal is ready for the Final Review. If the SIP Editors do not receive a response from the SIP author then the SIP Editors may at their discretion either close the pull request or move the proposal on for a Final Review.
+
+* If agreeable, an SIP Editor will assign the SIP a number (generally the number of the SIP pull request), change the status of the SIP pull request from "draft" to "ready", and move the SIP on for a Final Review.  
+* Reasons for denying an SIP number and closing the pull request include the SIP being too unfocused, too broad, duplication of effort, being technically unsound, not providing proper motivation or addressing legitimate concerns by reviewers, or not in compliance with SIP-X.  
+
+**Final Review**
+
+During the Final Review, the proposal SHOULD NOT be changed. This gives Sovryn Voters a chance to review the proposal in its final state before the SIP is voted on.
+
+### Stage V: Sovryn Vote
+
+After a SIP has gone through its Final Review, a SIP Editor will add a "Ready to vote" tag to the pull request. At the soonest available opportunity, Sovryn Guardians will review proposals that have the "Ready to vote" tag and decide whether or not they will veto the proposal.
+
+Once a proposal has been tagged "Ready to vote", the SIP author MAY submit the proposal to the Sovryn Governance Dashboard for a vote by Sovryn Voters. Anyone else MAY submit the proposal instead but they SHOULD make an effort to communicate with the SIP author first to make sure the proposal is ready to be submitted and will not be accidentally submitted multiple times.
```

Comment:
the SIP author may not be able to create the proposal himself, because proposal creation is limited to addresses with at least 1% (delegated) voting power. Therefore, someone with sufficient voting power should be in charge of doing it.

Reply:
@ororopickpocket Yes I tried to keep this part worded somewhat vaguely so as to not be overly prescriptive regarding how proposals are submitted. I deliberately chose not to mention the parameters of the proposal submission requirements (or the parameters of the vote themselves) since these are defined elsewhere and could change -- and I see no need to have to update this document every time there's a simple parameter change on the governance contracts. 

That said, you bring to mind a good idea, which is that we may want to point to the governance contract documentation somewhere in this document (perhaps in this section you're commenting on) once it's ready, to at least let proposal authors reading this know that they should be aware of the governance contract requirements for submitting and passing a proposal onchain.